### PR TITLE
Add documentation for Point alarm_control_panel

### PR DESCRIPTION
**Pull request by fredrike**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20972

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/point.markdown [hunk 1]
```diff
@@ -11,8 +11,9 @@ logo: minut.svg
 ha_category: Hub
```
Comment (klaasnicolaas) [inline source/_components/point.markdown]:
```suggestion
ha_category: 
  - Hub
  - Alarm
  - Binary Sensor
  - Sensor
```

You can merge all information from the `Alarm`, `Binary Sensor` and `Sensor` page and put it on the `Hub` page and then delete the pages.

File: source/_components/point.markdown [hunk 2]
```diff
@@ -11,8 +11,9 @@ logo: minut.svg
...
 featured: false
 ha_release: "0.83"
+ha_config_flow: true
 ha_iot_class: "Cloud Polling"
-ha_qa_scale: silver
+ha_qa_scale: gold
 ---
```
Comment (klaasnicolaas) [inline source/_components/point.markdown:24]:
```suggestion
redirect_from:
  - /components/binary_sensor.point/
  - /components/sensor.point/
---
```

Comment (klaasnicolaas) [inline source/_components/point.markdown:24]: Add also on this page then:
```suggestion
There is currently support for the following device types within Home Assistant:
  - Alarm
  - Binary Sensor
  - Sensor
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/point.markdown (hunk 2 above)
Comment (fredrike) [reply source/_components/point.markdown:24]: Where should I place this text?

File: source/_components/point.markdown [hunk 3]
```diff
@@ -2,20 +2,33 @@
 layout: page
 title: "Minut Point"
 description: "Instructions on how to integrate Minut Point into Home Assistant."
-date: 2018-11-19
```
Comment (klaasnicolaas) [inline source/_components/point.markdown]: Please revert this to: `2018-11-19`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/point.markdown [hunk 4]
```diff
@@ -2,20 +2,33 @@
...
+date: 2019-02-12
```
Comment (fredrike) [inline source/_components/point.markdown]:
```suggestion
date: 2018-11-19
```

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by fredrike** [comment]
@klaasnicolaas, the home-assistant/home-assistant#20972 have been merged now..

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ We can merge this PR, will be released in the 0.89 release.

**Pull request merged**